src/main/java/com/sfjs/crud/entity/AccountEntity.java

```diff
@@ -50,4 +50,10 @@ public class AccountEntity extends BaseEntity {
   @JsonIgnore
   @OneToOne(mappedBy = "account", optional = true)
   private FellowEntity fellow;
+
+  @Getter
+  @Setter
+  @JsonIgnore
+  @OneToMany(mappedBy = "account")
```

Comment:
Does this mapping mean that the account can have several tokens?
Or can the token belong to several accounts?

Either way, is this expected?

Reply:
This is expected.  The account can have multiple tokens.  This will happen if the account holder generates a new token multiple times.  Tokens expire and can only be used once.

Reply:
Got it, thanks!